integration-test/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.1"
```

Comment:
is this file in root still required? https://github.com/newrelic/nri-flex/blob/master/docker-compose.yml

Reply:
As long as BuildKite is activated, yes. We will remove it when we remove the BuildKite tests.